doc/ref_model/chapters/chapter08.rst

```diff
@@ -210,37 +271,213 @@ The API and Cloud Brokerage models are illustrated in the following diagrams:
--  provider of cloud services, through operation of their own cloud or reselling of cloud services, in which case the typical consideration is margin (cost to offer services vs income received).
+-  consumer, in which case the typical consideration is total cost of ownership as the consumption is to usually to support internal business 
+operations: BSS/OSS systems;
+-  provider of cloud services, through operation of their own cloud or reselling of cloud services, in which case the typical consideration is 
+margin (cost to offer services vs income received).
 
 These two stances will drive differing approaches to how a CSP would look to manage how it interacts within a Multi-Cloud environment.
 
+Hybrid, Edge, and Multi-Cloud unified management Platform 
+~~~~~~~~~~~~~~~~~~~~
+
+As organisations spread their resources across on-premises, multiple clouds, and the Edge, the need for a single set
+of tools and processes to manage and operate across these Hybrid, Edge, and Multi-clouds (HEM clouds) is obvious 
+as can be seen from the following simplistic scenarios. 
+
+Scenario: An operator has private clouds that it utilises for its workloads. Over time, the operator evolves their environment:
+
+- A: The operator has decided to utilise one or more public clouds for some of its workloads.
+- B: The operator has decided to utilise an edge cloud for some of its clients.
+- C: The operator has decided to create edge clouds for some of its clients.
+
+Scenario B can be treated as being the same as Scenario A. Scenario C is akin to the private cloud except for location
+and control over the facilities at that location. For its workloads, the operator will have to utilise the target clouds 
+tools or APIs to create the necessary accounts, billing arrangements, quotas, etc. Then create the needed resources, 
+such as VMs or Kubernetes clusters, etc. Following up with creating needed storage, networking, etc. before 
+onboarding the workload and operating it. This is complex even when the operator is dealing with say only one other 
+cloud, in addition to operating its own cloud. The operator is faced with a number of challenges including acquiring a 
+new set of skills, knowledge of APIs, tools, and the complexity of managing different policies, updates, etc. This 
+becomes impossible to manage when incorporating more than one other cloud. Hence the need for a Single Pane of Glass.
+
+This Hybrid, Edge, and Multi-Cloud unified management Platform (HEMP) (a.k.a. Single-Pane-of-Glass) provides capabilities to consistently:
+
+-  manage accounts, credentials, resources and services
+  -  across facilities (regions, data centers, edge locations)
+-  interoperate the different clouds
+-  implement common policies and governance standards
+-  manage a common security posture
+-  provide an integrated visualisation into the infrastructure and workloads.
+
+through a common set of governance and operational practices.
```

Comment:
@joaoarodrigues The common set of governance and operational refers to the constituent clouds. So any capabilities provided by constituent clouds can be managed using common terms and methods rather than through the different terms and methods used by constituent cloud methods. It is not "common" for all "Teams".